tests/load-tests.php

```diff
@@ -97,6 +97,32 @@ function( $default, $option, $passed_default ) use ( &$has_passed_default ) {
 		$this->assertSame( $new_value, $settings );
 	}
 
+	/**
+	 * @dataProvider data_legacy_modules
+	 */
+	public function test_legacy_module_for_perflab_get_module_settings( $legacy_module_slug, $current_module_slug ) {
+		$new_value = array(
+			'site-health/audit-autoloaded-options' => array( 'enabled' => true ),
+			'site-health/audit-enqueued-assets'    => array( 'enabled' => true ),
+			'site-health/audit-full-page-cache'    => array( 'enabled' => true ),
+			'site-health/webp-support'             => array( 'enabled' => true ),
+		);
```

Comment:
You can simplify this, you don't need to re-include the list of legacy module slugs here again. Only working with the single module that is being tested is sufficient.
```suggestion
		$new_value = array( $legacy_module_slug => array( 'enabled' => true ) );
```